composer.json

```diff
@@ -63,7 +63,8 @@
         "friendsofphp/php-cs-fixer": "^2.16.2",
         "ezsystems/ezplatform-code-style": "^0.1",
         "phpunit/phpunit": "^8.2",
-        "matthiasnoback/symfony-dependency-injection-test": "^4.1"
+        "matthiasnoback/symfony-dependency-injection-test": "^4.1",
+        "symfony/phpunit-bridge": "^5.1"
```

Comment:
How is this related to the PR?

Reply:
I use `ClockMock` for `time()` mocking in tests.